app/src/main/java/woowacourse/movie/movielist/MovieRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package woowacourse.movie.movielist
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import woowacourse.movie.Ad
+import woowacourse.movie.R
+import woowacourse.movie.movie.Movie
+
+class MovieRecyclerViewAdapter(
+    private val movies: List<Movie>,
+    private val ad: Ad,
+    private val movieOnItemClicked: (Int) -> Unit,
+    private val adOnItemClicked: (Ad) -> Unit,
+) :
+    RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
```suggestion
) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```
사소하지만 콜론과 같은 라인에 작성해주시는 것이 컨벤션입니다.